ytest/auth.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 The GoPlus Authors (goplus.org). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ytest
+
+import "net/http"
+
+type JwtAuth struct {
+	// Type     string
+	JwtToken string
+}
+
+type JwtRoundTripper struct {
+	RoundTripper http.RoundTripper
+	Token        string
+}
+
+func (rt *JwtRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Set("Authorization", rt.Token)
```

Comment:
according implementation, there is nothing about `jwt`.
so please change to another class name.

Reply:
Thanks for your comments. JwtAuth is already renamed to tokenAuth.